Tasks/SonarQubePostTest/SonarQubePostTest.ps1

```diff
@@ -12,13 +12,11 @@ ExitOnPRBuild
 . $PSScriptRoot/PRCA/Orchestrator.ps1
 
 . $PSScriptRoot/SonarQubeMetrics.ps1
-. $PSScriptRoot/SonarQubeReportHandler.ps1
+. $PSScriptRoot/SummaryReport/ReportBuilder.ps1
```

Comment:
I split the summary report logic into multiple files and placed them into a SummaryReport directory